tests/evm/test_callop.py

```diff
@@ -257,7 +257,9 @@ def test_callop(
         is_empty_code_hash = True
     callee_bytecode_hash = Word(callee_bytecode_hash if not callee.is_empty() else 0)
 
-    is_success = False if callee is CALLEE_WITH_REVERT_BYTECODE else True
+    # Only check balance and stack depth
+    is_precheck_ok = caller.balance >= value and depth < 1025
+    is_success = False if is_precheck_ok is False or callee is CALLEE_WITH_REVERT_BYTECODE else True
```

Comment:
The logic can be simplified.

```suggestion
    is_success = (is_precheck_ok and callee != CALLEE_WITH_REVERT_BYTECODE)
```